### PR TITLE
podman_image: Fix test on JeOS, add to Leap 15.1+ as well

### PR DESCRIPTION
**Pull request by Vogtinator**
Adding two workarounds...

Verification runs:
Tumbleweed JeOS: http://10.160.67.86/tests/373 (fails, product fixed with https://build.opensuse.org/request/show/723536)
Leap 15.1 JeOS: http://10.160.67.86/tests/376

### CODE REVIEW

File: tests/console/podman_image.pm [hunk 1]
```diff
@@ -25,6 +25,17 @@ sub run {
 
     zypper_call "in podman";
 
+    # If there is no other CNI configured...
+    if (get_var("CNI", "podman") == "podman") {
```
Comment (sysrich) [inline tests/console/podman_image.pm]: don't you mean this to be a check_var?
`
if checkvar("CNI", "podman")`

File: tests/console/podman_image.pm [hunk 2]
```diff
@@ -25,6 +25,17 @@ sub run {
...
+        # ... use the minimal podman CNI
+        zypper_call "in podman-cni-config";
+    }
+
+    if (is_leap("=15.1")) {
```
Comment (sysrich) [inline tests/console/podman_image.pm]: So we wont need to install the parser on 15.2 or later?
What about SLE?

Comment (Vogtinator) [reply tests/console/podman_image.pm]: According to the bug report it's stuck in SLE 15 SP1 maintenance.

File: tests/console/podman_image.pm (hunk 1 above)
Comment (Vogtinator) [reply tests/console/podman_image.pm]: No, that would do something else entirely.

**Pull request merged**